source/location_access/locacc_death_mountain.cpp

```diff
@@ -491,40 +523,77 @@ void AreaTable_Init_DeathMountain() {
-                                             },
-                                              /*Glitched*/
-                                              [] {
-                                                  return IsChild && CanSurviveHeatFor(-1, 24) && HasBombchus &&
-                                                         CanDoGlitch(GlitchType::BombHover, GlitchDifficulty::NOVICE);
-                                              } }),
-                 Entrance(DMC_UPPER_NEARBY, { [] { return IsAdult && CanPlantBean(DMC_CENTRAL_LOCAL); } }),
-                 Entrance(FIRE_TEMPLE_ENTRYWAY, { [] {
-                                                     return (IsChild && CanSurviveHeatFor(-1, 24) &&
-                                                             ShuffleDungeonEntrances.IsNot(SHUFFLEDUNGEONS_OFF)) ||
-                                                            (IsAdult && CanSurviveHeatFor(24));
-                                                 },
-                                                  /*Glitched*/
-                                                  [] {
-                                                      return CanDoGlitch(GlitchType::ASlide,
-                                                                         GlitchDifficulty::INTERMEDIATE) &&
-                                                             CanSurviveHeatFor(48);
-                                                  } }),
-             });
+    areaTable[DMC_CENTRAL_LOCAL] = Area(
+        "DMC Central Local", "Death Mountain Crater", DEATH_MOUNTAIN_CRATER, NO_DAY_NIGHT_CYCLE,
+        {
+            // Events
+            EventAccess(&BeanPlantFairy, { [] {
+                return BeanPlantFairy || (CanPlantBean(DMC_CENTRAL_LOCAL) && CanPlay(SongOfStorms));
+            } }),
+        },
+        {
+            LocationAccess(DMC_LOWER_RED_RUPEE_1, { [] { return IsChild && CanSurviveHeatFor(8, 24); } }),
+            LocationAccess(DMC_LOWER_RED_RUPEE_2, { [] { return IsChild && CanSurviveHeatFor(8, 24); } }),
+            LocationAccess(DMC_LOWER_BLUE_RUPEE_1, { [] { return IsChild && CanSurviveHeatFor(8, 24); } }),
+            LocationAccess(DMC_LOWER_BLUE_RUPEE_2, { [] { return IsChild && CanSurviveHeatFor(8, 24); } }),
+            LocationAccess(DMC_LOWER_BLUE_RUPEE_3, { [] { return IsChild && CanSurviveHeatFor(8, 24); } }),
+            LocationAccess(DMC_LOWER_BLUE_RUPEE_4, { [] { return IsChild && CanSurviveHeatFor(8, 24); } }),
+            LocationAccess(DMC_LOWER_BLUE_RUPEE_5, { [] { return IsChild && CanSurviveHeatFor(8, 24); } }),
+            LocationAccess(DMC_LOWER_BLUE_RUPEE_6, { [] { return IsChild && CanSurviveHeatFor(8, 24); } }),
+            LocationAccess(DMC_SCARE_CROW_RUPEE_CIRCLE_GREEN_RUPEE_1, { [] {
+                               return IsAdult && CanSurviveHeatFor(24, -1) &&
+                                      (CanUse(DistantScarecrow) || Here(DMC_UPPER_LOCAL, { [] { return IsAdult; } }));
```

Comment:
While it's possible to jump down to the platform damageless, would suggest adding a logic trick as is not obvious and otherwise consider it to be damaging
```suggestion
                                      (CanUse(DistantScarecrow) || Here(DMC_UPPER_LOCAL, { [] { return IsAdult && (<eg: LogicDMCPlatformJump> || EffectiveHealth > 2 || Fairy); } }));
```